MINIKUBE.md

```diff
@@ -0,0 +1,170 @@
+# local development with minikube
+
+1. clone and run [razeedash](https://github.com/razee-io/Razeedash.git)
+
+```bash
```

Comment:
Needs to be indented 2 tabs so ordered number steps will increment on the markdown